examples/Boost-math/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2013, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-boost)
+
+# download boost
+hunter_add_package(Boost COMPONENTS math)
+
+# now boost can be used
+# you need to choose which form of Boost::math you want
```

Comment:
Can we use all of them simultaneously?

Reply:
Let's link all of them. By this we will tests that all `Boost::math_*` targets are valid.